charts/team-ns/templates/builds/buildpack.yaml

```diff
@@ -43,7 +43,7 @@ spec:
 apiVersion: tekton.dev/v1beta1
 kind: PipelineRun
 metadata:
-  name: buildpacks-build-{{ .name }}
+  name: buildpacks-build-{{ .name }}-{{ regexReplaceAll "[|.]{1}" .tag "-" }}
```

Comment:
I looked into the schema and see that the tag name can be any string that does not contain white characters
```
imageTag:
    pattern: '[\w][\w.-]{0,127}'
```
So for example `my_tag` would be also applicable. However k8s would have reject this name as it is not matching naming pattern. Could you improve this regex?

Reply:
underscores are replaced by "-" now